### ISSUE TITLE
V3.8.0 Can the separate crop and rotate buttons be merged again into 1 button?

### ISSUE THREAD

**Issue by JvBoven** [darktable-org/darktable#11035]
**Is your feature request related to a problem? Please describe.**
In previous darktable versions, the crop and rotate functions were combined into 1 button. 
In V3.8.0 (I really appreciate the .cr3 support!), crop and rotate were separated into 2 buttons.
In my workflow for non-level photos, I now have to switch often between those 2 buttons: rotate, crop, mmmmm not working, back to rotate, rotate a  bit more/less, back to crop, etc

**Describe the solution you'd like**
Is it possible to get the crop and rotate back into 1 button please?

**Alternatives**
If the current 2 buttons will remain, as I can imagine that there is a good reason for it; can there be a 3rd buttons added with both crop and rotate functions combined again?

**Additional context**

![crop-rotate](https://user-images.githubusercontent.com/98693806/151701039-2a72d108-1f13-4974-badc-b5911f582bf0.png)

**Comment by elstoc** [comment]
I understand and I share your concerns to some extent but from a processing point of view these two operations are better placed at different points in the pipeline. In this case, the operations that might distort the image are placed together in a single module, before the retouch module so that the retouch module includes those correcting distortions. The crop module is placed after the retouch module so that the retouch module may access parts of the image outside of the cropped area. Placing operations that require interpolation (perspective and rotation) in the same module also has advantages for quality, I believe.

The split was done in order to correct these issues as well as to remove the crop & rotate module which was becoming increasingly difficult to develop, due to the many things it was trying to do at the same time.

There are, however, ways to improve your efficiency using a combination of keyboard shortcuts and/or the quick access panel. Personally I have a keyboard shortcut to toggle the focus of the crop module and another to toggle the focus of the rotate & perspective module. This way I can activate both modules quickly. Another option would be to assign a shortcut to just the rotation aspect of the rotate & perspective module allowing you to rotate while crop is active.

**Comment by ptilopteri** [comment]
why not take advantage of the option(s) to apply crop within
rotate&perspective?  you can set rot/per automagic crop to 
original format or largest are.  

you can always crop again after you are satisfied with the rot/per
module.

I assign hot-keys so switching doesn't appear to take as much
effort.  fwiw: I also preferred them in the same module at first,
but I had been using them that way for many years.

if this satisfies your wants, please consider closing the issue.

**Comment by JvBoven** [comment]
Hi @ptilopteri 
I used both 'largest' and 'original'; 95% of the time these work fine.
But today I had some photos where a 3rd option like 'freehand' would be nice in that rotate option. Due to obstructions in the margins, I could not use 'largest' and 'original' to get my desired cropping. I ended up with rotating first and freehand crop later, to end up with my required results.
![crop-rotate1](https://user-images.githubusercontent.com/98693806/151704417-fda393b3-35c7-4cb3-a803-41795879c83e.png)

But the ticket can be closed as these kind of photos are 'edge cases' that only happen very rare.
